ansible/roles/kubelet/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+#Default Override-able variables for the Kubelet role
+cni_version: "v0.4.0"
```

Comment:
It's good ansible practice to have all your variable in the role defaults, so that roles can deployed independently of the rest of the infrastructure.  For example, if in the future a developer wanted to use this role, they could easily look into the defaults/main.yml to see what the default values are for the role and override them in the inventory, groupvars, or hostvars.   Having all your variables laid out in the defaults shows the developer which variables are available to the role and how they can override them. You can read more about it here:  http://docs.ansible.com/ansible/playbooks_variables.html